.gitignore

```diff
@@ -25,3 +25,7 @@ app/test/bad_block_*
 # do not include downloaded embedded binaries
 internal/embedding/**
 **/*token.sh
+# ignore talis files
+app.toml
+config.toml
+config.json
```

Comment:
no changes needed I don't think, but the problems you hit might just have been from running talis in celestia-app repo, where as I always run it outside where each experiment has its own repo

Reply:
yeah it lets you specify the source root but "src" + "importPath" is hardcoded where it might not match everyone's structure. I don't want to die on this hill though so we can just leave it as is.

Reply:
it is a good point and was fixed in target branch to not have "celestiaorg". currently it will only have celestia-app, so we must point to where the celestia-app repo is